app/scripts/lib/transaction/decode/util.ts

```diff
@@ -121,7 +121,7 @@ function normalizeDecodedParamValue(value: any): any {
   const hexValue = value._hex;
 
   if (hexValue) {
-    return parseInt(hexValue, 16);
+    return BigInt(hexValue).toString();
```

Comment:
The tokenId value returned by `useDecodedTransactionData` was in scientific format and different from the tokenId format displayed in NFT details page.

`parseInt(hexValue, 16)` converted the hexadecimal string to a base-10 number.
`parseInt("0x681bd29732282184a07b713b2e79b2c439f6daaeb5c008bd4ad4ac889d4a4687", 16);`

instead using:
`BigInt(hexValue) `correctly converts the large hexadecimal number into a BigInt.